pyaedt/hfss.py

```diff
@@ -47,7 +49,14 @@ class Hfss(FieldAnalysis3D, object):
         designs are present, an empty design is created.
     solution_type : str, optional
         Solution type to apply to the design. The default is
-        ``None``, in which case the default type is applied.
+        ``None``, in which case the user-defined
+        default type is applied.
+        Options are:
+        - "Terminal"
+        - "Modal"
+        - "SBR+"
+        - "Transient"
+        - "Eigenmode"
```

Comment:
```suggestion
        - "Eigenmode"

```
Blank lines need to appear before first list item and after last one.